### ISSUE TITLE
Bug: Override custom titlebar settings independent of colorTab.titleBackground 

### ISSUE THREAD

**Issue by tjarbo** [oreporan/color-tabs-vscode#40]
Hey :wave:,
first of all: Thank you very much for this really helpful extentsion 😄. Unfortunatly I've found a little bug.

## Description
The extention overwrites my own `workbench.colorCustomizations`. I have set up different titlebar customizations to sepreate multiple vscode instanzes (like frontend/backend or Projekt 1/Projekt 2). After I have activated the extension, the properties `titleBar.activeBackground` and `titleBar.activeForeground` will overwrite. The setting `colorTabs.titleBackground` is set to false.

## Steps to reproduce
1. In _settings.json_ (workspace)
```json
{
 "workbench.colorCustomizations": {
 "titleBar.activeForeground": "#000",
 "titleBar.inactiveForeground": "#000000CC",
 "titleBar.activeBackground": "#F3DB04",
 "titleBar.inactiveBackground": "#f3db04bd"
 }
}
```
2. Setup the extension as known + 
```json
{
"colorTabs.titleBackground": false
}
```
3. Open a file matching the regex

## What is expected?
The color of the statusbar changed - the titlebar color is `#F3DB04`

## What happens instead?
The color of the statusbar changed - the titlebar color is reset to the theme color.

**Comment by oreporan** [comment]
Hey
yes you're right
Let me make a fix

**Comment by oreporan** [comment]
Released in 0.11.0

**Comment by tjarbo** [comment]
Great! Thank you very much!
